src/e2e/iOS/__tests__/caret.ts

```diff
@@ -124,3 +124,46 @@ it('Tap hidden uncle', async () => {
   const selectionTextContent = await mobileBrowser.execute(() => window.getSelection()?.focusNode?.textContent)
   expect(selectionTextContent).toBe('d')
 })
+
+it('Tap empty content while keyboard up', async () => {
+  const importText = `
+    - a
+      - b
+        - c
+      - d`
+
+  await gesture(mobileBrowser, gestures.newThought)
+  await paste(mobileBrowser, [''], importText)
+  await clickThought(mobileBrowser, 'b')
+  await clickThought(mobileBrowser, 'c')
+
+  const editableNodeHandleD = await waitForEditable(mobileBrowser, 'd')
+  await tapWithOffset(mobileBrowser, editableNodeHandleD, { x: 20, y: 200 })
+
+  // Wait until cursor change
+  await mobileBrowser.waitUntil(async () => await getEditingText(mobileBrowser) === 'b')
+  const editingText = await getEditingText(mobileBrowser)
+  expect(editingText).toBe('b')
+})
+
+it('Tap empty content while keyboard down', async () => {
+  const importText = `
+    - a
+      - b
+        - c
+      - d`
+
+  await gesture(mobileBrowser, gestures.newThought)
+  await paste(mobileBrowser, [''], importText)
+  await clickThought(mobileBrowser, 'b')
+  await clickThought(mobileBrowser, 'c')
+  await hideKeyboardByTappingDone(mobileBrowser)
+
+  const editableNodeHandleD = await waitForEditable(mobileBrowser, 'd')
+  await tapWithOffset(mobileBrowser, editableNodeHandleD, { x: 20, y: 200 })
+
+  // Wait until cursor change
+  await mobileBrowser.waitUntil(async () => await getEditingText(mobileBrowser) === 'b')
+  const editingText = await getEditingText(mobileBrowser)
+  expect(editingText).toBe('b')
```

Comment:
We should also test that the keyboard has stayed down, i.e. there is no focusNode.

Reply:
@raineorshine, I made changes.
After tapping the empty area while the keyboard is down, the focusNode is `<span role="presentation" class="Resizer vertical "></span>`  element and it isn't null. That is why I used isKeyboardShown function.